tools/codegen/context.py

```diff
@@ -81,15 +82,6 @@ def wrapper(f: F, backend_index: BackendIndex) -> T:
 
 # Convenience decorator for functions that explicitly take in a BackendIndex,
 # instead of indirectly taking one in as a closure
-def with_native_function_and_selector_and_index(
```

Comment:
The comment above should go as well no?

Reply:
The comment also applies to the wrapper below the one that I deleted (technically it takes a dict of BackendIndices, I can fix that)